messages.pot

```diff
@@ -0,0 +1,101 @@
+# Translations template for PROJECT.
+# Copyright (C) 2021 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2021-05-04 17:09-0700\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.9.0\n"
+
+#: microsetta_interface/implementation.py:379
+msgid "Unable to validate the kit name; please reload the page."
+msgstr ""
+
+#: microsetta_interface/implementation.py:398
+msgid ""
+"The provided kit id is not valid or has already been used; please re-"
```

Comment:
is it weird this is on a newline?

Reply:
I think it likes breaking at 80 characters due to the defaults in babel.cfg, so it saw the - and went for it.

Reply:
cool cool